pkg/network/tracer/tracer_test.go

```diff
@@ -1150,7 +1151,7 @@ func (s *TracerSuite) TestUnconnectedUDPSendIPv4() {
 	cfg := testConfig()
 	tr := setupTracer(t, cfg)
 
-	remotePort := rand.Int()%5000 + 15000
+	remotePort := testutil.GetOpenPortUDP(t)
```

Comment:
I think this is another test where using `rand` should be ok, since the `ListenUDP` call on line 1157 should allocate a random source port. The destination port shouldn't really have a conflict since there are no other sockets with the same source, dest port combination. And this is UDP so we don't really need to be talking to another server at the other end.

Reply:
Gotcha, can the same be said for IPv6 version of this test below?

Reply:
Yes, I think so.